alerter/NetworkAlerter.cs

```diff
@@ -0,0 +1,17 @@
+
+using System;
+
+namespace AlerterSpace
+{
+ public class NetworkAlerter : INetworkAlerter

```

Comment:
This class is not used anywhere. Is it intended to be used or unused. If unused then the code should be removed.